<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtSa_Dialog_Limit</class>
 <widget class="QWidget" name="RtSa_Dialog_Limit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/***********************indoor*****************/
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#widget
{
background-color:rgba(211,211,211,0.9);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
	color:rgb(130, 130, 130);
	border-style:flat;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_LimitTest,
#pb_Edge,
#pb_toggle
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_LimitTest:checked,
#pb_Edge:checked,
#pb_toggle:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/rtsa/image/toggle_on.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_LimitTest:!checked,
#pb_Edge:!checked,
#pb_toggle:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/rtsa/image/toggle_off.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Upper:checked
{
	background-color:transparent;
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Upper:!checked
{
background-color:transparent;
padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}

#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Lower:checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Lower:!checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#widget&gt;QLabel
{
font: 75 13px &quot;Arial&quot;;
color:rgb(46,46,46);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
RtSa_LineEdit
{
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
RtSa_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
	padding:6px 6px 5px 8px ;
}
/***********************outdoor*****************/
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#widget
{
background-color:rgba(211,211,211,0.9);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
	color:rgba(0, 0, 0,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_LimitTest,
#pb_Edge,
#pb_toggle
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_LimitTest:checked,
#pb_Edge::checked,
#pb_toggle:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/rtsa/image/toggle_on.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_LimitTest:!checked,
#pb_Edge:!checked,
#pb_toggle:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/rtsa/image/toggle_off.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Upper:checked
{
	background-color:transparent;
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Upper:!checked
{
background-color:transparent;
padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Lower:checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Lower:!checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
RtSa_LineEdit
{
	background-color: rgba(255, 255, 255, 0.6);
	color:rgb(161, 114, 23);
	padding-left:6px;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
RtSa_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
/***********************night*****************/
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#widget
{
background-color:rgba(208,182,139,0.9);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
border-image: url(:/rtsa/image/night/btn_close_normal.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_cancel:pressed
{
border-image: url(:/rtsa/image/night/btn_close_click.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_LimitTest,
#pb_Edge,
#pb_toggle
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_LimitTest:checked,
#pb_Edge::checked,
#pb_toggle:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/rtsa/image/toggle_on.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_LimitTest:!checked,
#pb_Edge:!checked,
#pb_toggle:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/rtsa/image/toggle_off.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_Upper:checked
{
	background-color:transparent;
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_Upper:!checked
{
background-color:transparent;
padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_Lower:checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_Lower:!checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
RtSa_LineEdit
{
	background: rgba(251,220,167,0.60);
   color:rgba(159,98,0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_Limit[status  =  &quot;night&quot;] 
RtSa_LineEdit:disable
{
	color:rgba(133,133,133,0.6);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
RtSa_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}</string>
  </property>
  <property name="status" stdset="0">
   <string>night</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>648</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Limit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>212</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_toggle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>281</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#7B7B7B;</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LM1">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>71</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/outdoor/btn_bw.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} </string>
    </property>
    <property name="text">
     <string>Limit1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LM3">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>137</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/outdoor/btn_bw.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} </string>
    </property>
    <property name="text">
     <string>Limit3</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LM4">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>170</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/outdoor/btn_bw.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} </string>
    </property>
    <property name="text">
     <string>Limit4</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LM2">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>104</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/outdoor/btn_bw.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} </string>
    </property>
    <property name="text">
     <string>Limit2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Edge">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>391</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#7B7B7B;</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_DeleteAll">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>592</y>
      <width>216</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_DeleteAll
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(255, 255, 255, 1);	
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_DeleteAll:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_DeleteAll
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(0, 0, 0, 1);	
	background-image: url(:/rtsa/image/outdoor/btn_w216_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_DeleteAll:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_DeleteAll
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: #FBDBA7;
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_DeleteAll:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}
</string>
    </property>
    <property name="text">
     <string>Delete all limits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LM6">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>236</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/outdoor/btn_bw.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} </string>
    </property>
    <property name="text">
     <string>Limit6</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>284</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Upper">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>540</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#7B7B7B;</string>
    </property>
    <property name="text">
     <string>Upper</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">LimitTypeGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>492</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Lower">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#7B7B7B;</string>
    </property>
    <property name="text">
     <string>Lower</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">LimitTypeGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pb_LimitTest">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>490</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="leMargin">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>440</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LM5">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>203</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/outdoor/btn_bw.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} 

#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
    text-align:left;
    padding-left:10px;
   background-image: url(:/rtsa/image/btn_mkr_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
QPushButton:checked
{ 
  background-image: url(:/rtsa/image/btn_mkr_click.png);
} </string>
    </property>
    <property name="text">
     <string>Limit5</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>537</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_EditMask">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>331</y>
      <width>216</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_EditMask
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(255, 255, 255, 1);	
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;indoor&quot;]
#pb_EditMask:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_EditMask
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(0, 0, 0, 1);	
	background-image: url(:/rtsa/image/outdoor/btn_w216_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_EditMask:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_EditMask
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: #FBDBA7;
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_Limit[status = &quot;night&quot;]
#pb_EditMask:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}</string>
    </property>
    <property name="text">
     <string>Edit Limit Line Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>393</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit Edge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Limit
Edge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RtSa_LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">rtsa_lineedit.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="LimitTypeGroup"/>
 </buttongroups>
</ui>
